cmd/agent/config/env.go

```diff
@@ -0,0 +1,94 @@
+package config
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"github.com/caarlos0/env/v6"
+	"log"
+	"strconv"
+	"strings"
+	"time"
+)
+
+// PollInterval Интервал между сборкой данных
+var PollInterval = time.Duration(2) * time.Second
+
+// ReportInterval Интервал между отправкой данных
+var ReportInterval = time.Duration(10) * time.Second
+
+// ServerURL Url сервера получателя метрик
+var ServerURL = "http://localhost:8080"
```

Comment:
 Значения по-умолчанию можно объявить как константы. Не обязательно использовать `time.Duration(10)`, так как умножение `2 * time.Second`  так же даст тип `Duration`.

```suggestion
const (
	// PollInterval Интервал между сборкой данных
	PollInterval = 2 * time.Second

	// ReportInterval Интервал между отправкой данных
	ReportInterval = 10 * time.Second

	// ServerURL Url сервера получателя метрик
	ServerURL = "http://localhost:8080"
)
```

Reply:
В константах убрал приведение к Duration